docker/candlepin-base/cp-test

```diff
@@ -520,12 +531,15 @@ if [ "$DEPLOY" == "1" ]; then
     sleep 7
 fi
 
-if [ "$RSPEC" == "1" ]; then
-    echo "Running rspec tests..."
+if [ "$SPEC_TESTS" == "1" ]; then
+    echo "Running spec tests..."
     CLEAN_CP=1
 
+    tee /var/log/candlepin/spec.log < /tmp/teepipe &
+    build_spec $SPEC_FILTER
```

Comment:
don't we need to redirect to /tmp/teepipe here?

Reply:
Fixed

Reply:
The spec.log still isn't collected. Looking at the `collect_artifacts()` method, you need to add spec.log in the list of `CANDLEPIN_LOGS` to collect. Also, another thing which I don't know if it makes a difference, but I see it used elsewhere here, is to remove and setup the teepipe again when you want to use it more than once (between spec and rspec):
``` 
rm -f /tmp/teepipe
mkfifo /tmp/teepipe
```

Reply:
Added spec.log to CANDLEPIN_LOGS.

Regarding the pipe, we're safe to reuse it. Just to be certain, I tried it manually to verify:
```
(08:26:51) crog@cl-vm-1:~/devel/candlepin-alt (crog/4.0/64bit_entity_versions) $ mkfifo /tmp/teepipe
(08:28:49) crog@cl-vm-1:~/devel/candlepin-alt (crog/4.0/64bit_entity_versions) $ tee a.txt < /tmp/teepipe &
[1] 187052
(08:29:07) crog@cl-vm-1:~/devel/candlepin-alt (crog/4.0/64bit_entity_versions) $ echo "hello" > /tmp/teepipe
hello
[1]+  Done                    tee a.txt < /tmp/teepipe
(08:29:15) crog@cl-vm-1:~/devel/candlepin-alt (crog/4.0/64bit_entity_versions) $ tee b.txt < /tmp/teepipe &
[1] 187064
(08:29:23) crog@cl-vm-1:~/devel/candlepin-alt (crog/4.0/64bit_entity_versions) $ echo "hello again" > /tmp/teepipe
hello again
[1]+  Done                    tee b.txt < /tmp/teepipe
(08:29:30) crog@cl-vm-1:~/devel/candlepin-alt (crog/4.0/64bit_entity_versions) $ cat a.txt
hello
(08:29:34) crog@cl-vm-1:~/devel/candlepin-alt (crog/4.0/64bit_entity_versions) $ cat b.txt
hello again
```